lib/ace/ext/papeeria/highlighter.coffee

```diff
@@ -27,10 +25,13 @@ define((require, exports, module) ->
                 rangeRight = new Range(0, 0, pos.right.row, pos.right.column + 1)
                 session.$bracketHighlightRight = session.addMarker(rangeRight, "ace_error-marker", "text")
         session.$highlightRange = pos
+        if pos.left && pos.right
+            if (pos.right.row - pos.left.row) > 70
```

Comment:
@dbarashev 
Because at standard scale, the editor 70 accommodates lines. I could only come up with a way to determine the distant brackets.